api4/cloud.go

```diff
@@ -387,6 +387,29 @@ func handleCWSWebhook(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.Err = nErr
 			return
 		}
+	case model.EventTypeSendFirstCloudAdminWelcomeEmail:
+		user, err := c.App.GetUserByUsername(event.CloudWorkspaceOwner.UserName)
```

Comment:
Recently we've introduced a new convention. All the `model.AppError` variables should be called `appErr` to differentiate them from regular golang errors that should be called `err`